dts/arm/nordic/nrf52840.dtsi

```diff
@@ -174,6 +174,22 @@
 			interrupt-names = "wdt";
 			label = "WDT";
 		};
+
+		cryptocell: crypto@5002a000 {
+			compatible = "nordic,nrf-cc310";
+			reg = <0x5002A000 0x1000>;
+			label = "CRYPTOCELL";
+			status = "ok";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			cc310: crypto@5002b000 {
```

Comment:
I think we should use cryptocell310 instead of cc310, cc310 is very clear when referenced elsewhere.

Reply:
'b' --> 'B'

Reply:
cc310 --> cryptocell310: OK, I can do that.
'b' --> 'B': I can't do that - the same problem as above.